inc/cleantalk-public.php

```diff
@@ -354,6 +354,19 @@ class_exists('Jetpack', false) &&
     if ( ! empty($_POST) && apbct_custom_forms_trappings() ) {
         add_action('init', 'ct_contact_form_validate', 999);
     }
+
+    /**
+     * Internal Forms - Sendinblue Integration https://wordpress.org/plugins/mailin/
+     */
+    if (
+        (int)$apbct->settings['forms__check_internal'] === 1
```

Comment:
I think this won't work if JS disabled in the web browser. When you find a solution then merge the request.

Reply:
DONE